src/Middleware/Diagnostics/src/README.md

```diff
@@ -0,0 +1,6 @@
+ASP.NET Core Diagnostics
+===
+
+## Development
+
+Diagnostics middleware like `DeveloperExceptionPage` uses compiled Razor views. After updating the `*.cshtml` file you must run the [RazorPageGenerator](https://github.com/aspnet/AspNetCore-Tooling/tree/master/src/Razor/src/RazorPageGenerator) tool to generate an updated compiled Razor view.
```

Comment:
Thanks for adding this! How exactly does one run this tool after editing the CSHTML?